lib/pure/asyncftpclient.nim

```diff
@@ -79,7 +79,13 @@ proc connect*(ftp: AsyncFtpClient) {.async.} =
     # 120 Service ready in nnn minutes.
     # We wait until we receive 220.
     reply = await ftp.expectReply()
-  assertReply(reply, "220")
+
+  # Handle 220 messages from the server
+  if reply.startsWith("220"):
+    assertReply(reply, "220")
+    while reply.continuesWith("-", 3): # handle multiline 220 message
```

Comment:
I have two for servers that responds as follow:
```
220- # 
220- # Hello Guest
220- # How are you?
220- # 
220 No Anonimous login allowed
```
The server put the dash after 220 to tell the client that is a multiline message. The last 220 message will have a space character instead, to tell that this is the last 220 message line.